packages/core/src/services/evaluationsV2/llm/serializeEvaluatedDocumentLog.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  buildConversation,
+  DocumentLog,
+  formatMessage,
+  EvaluatedDocumentLog,
+  Workspace,
+} from '@latitude-data/core/browser'
+import { serializeAggregatedProviderLog } from '../../documentLogs/serialize'
+import { database } from '../../../client'
+import { ProviderLogsRepository } from '../../../repositories'
+import { Result, TypedResult } from '../../../lib/Result'
+
+export async function serializeEvaluatedDocumentLog(
+  {
+    workspace,
+    documentLog,
+  }: {
+    workspace: Workspace
+    documentLog: DocumentLog
+  },
+  db = database,
+): Promise<TypedResult<EvaluatedDocumentLog, Error>> {
+  const providerLogsScope = new ProviderLogsRepository(workspace.id, db)
+  const providerLogsResult = await providerLogsScope.findByDocumentLogUuid(
```

Comment:
presenters should always be pure lift this out of here

Reply:
Ok